extensions/amp-story/1.0/amp-story.css

```diff
@@ -984,3 +992,10 @@ amp-story-page .i-amphtml-story-page-open-attachment-label {
   text-shadow: 0px 0px 6px rgba(0, 0, 0, 0.36) !important;
   white-space: nowrap !important;
 }
+
+amp-story-quiz {
+  width: auto !important;
+  height: auto !important;
+  min-width: 224px !important;
```

Comment:
Plz add a small comment to explain that this is a narrow phone minus 48px on each side

Reply:
Done